content/core-features/authentication.textile

```diff
@@ -455,12 +455,11 @@ A revocation request must include one or more target specifiers which define the
 
 By default, supported target specifiers are:
 
-- clientId:<clientId> := For a string @clientId@, matches tokens that have a non-empty @clientId@ equal to that given.
-- revocationKey:<revocationKey> := Matches tokens that have a @revocationKey@ claim that matches the given @revocationKey@.
-
+- clientId := For a string @clientId@, matches tokens that have a non-empty @clientId@ equal to that given, for example: <code>targets: ["clientId:client1@example.com" ]</code>
+- revocationKey := Matches tokens that have a @revocationKey@ claim that matches the given @revocationKey@, for example: <code>targets: ["revocationKey:users.group1@example.com"]</code>
 The following target specifier can also be enabled for you by Ably, should your use case require it:
 
-- channel:<resourceName> := For a @resourceName@ that exactly matches one of the "resource names":/core-features/authentication#wildcards present in the token capabilities. Note that this is not the same thing as revoking all tokens that have access to the channel. For example, a token with a capability of @{"foo:*": ["*"]}@ will be revoked by a target of @channel:foo:*@, but a revocation to @channel:*:*@ will have no effect (even though that is a superset of the capabilities of @foo:*@), and nor will @channel:foo:bar@ (even for connections using the token to attach to that particular channel). It must be the exact string used in the token capabilities (which may be inherited from key capabilities).  
+- resourceName := For a @resourceName@ that exactly matches one of the "resource names":/core-features/authentication#wildcards present in the token capabilities. Note that this is not the same thing as revoking all tokens that have access to the channel. For example, a token with a capability of @{"foo:*": ["*"]}@ will be revoked by a target of @channel:foo:*@, but a revocation to @channel:*:*@ will have no effect (even though that is a superset of the capabilities of @foo:*@), and nor will @channel:foo:bar@ (even for connections using the token to attach to that particular channel). It must be the exact string used in the token capabilities (which may be inherited from key capabilities).  
```

Comment:
This change looks slightly odd compared to what you did in the previous section. The key is `channel` and the value is a resource name. We need to make that clearer. I think the description needs to be clarified.